Added a logs volume

rufuswilson opened the pull request:
I found myself having the need to pipe the output of commands to log files and I didn't want to clutter the output.
I added a logs volume and pipe the output of the installpkg, bash script running and slack build building to log files.

rufuswilson committed:
Added support for logs

rufuswilson committed:
Added logs volume

rufuswilson committed:
Added reference to logs

rufuswilson committed:
Added more dependency package types for installation

Lanjelin committed:
Update README.md

Formatting, wording

Lanjelin committed:
Update entrypoint.sh

Retain console output by using tee

Lanjelin commented:
By using tee, we retain console output, while having the option to export/mount the logs aswell.  
I'll merge this, unless you have anything more to add?

rufuswilson commented:
I tried to have a more radical approach: write to the output only the name of the steps and the command output would be spat in the appropriate log file.
My use case is the container run unattended.
In the case where it is run manually, I do see a point of having something printed back.
What if we had an environmental variable LOGONLY to prevent console output?

Lanjelin committed:
variable LOGONLY

Variable LOGONLY to prevent console output
Running the commands directly
https://github.com/Lanjelin/slackbuilder/issues/3

Lanjelin committed:
Update README.md

Update for LOGONLY

Lanjelin commented:
Added environment variable `LOGONLY` as suggested.  
The initial script was just a quick way to get stuff working, the commands are now executed directly and not though subshells. https://github.com/Lanjelin/slackbuilder/issues/3  

Thoughts?

rufuswilson commented:
It looks great!
(There is a typo in the readme for the logonly option in the suppress)

Lanjelin committed:
Update README.md

typo
